capabilities/loosely-coupled-teams.md

```diff
@@ -52,28 +52,28 @@ The following capabilities will be valuable for you and your team to explore, as
 
 ### [Code Maintainability](/capabilities/code-maintainability.md) - Downstream
 
 When teams operate independently, they can focus on maintaining and improving their own codebases. This autonomy fosters higher code quality and easier updates, as the impact of changes is localized and less likely to introduce systemic issues.
 
 ### [Empowering Teams To Choose Tools](/capabilities/empowering-teams-to-choose-tools.md) - Downstream
 
-The independence of loosely coupled teams enables each group to select tools that best meet their specific needs. This tailored approach can lead to increased productivity and innovation, as teams are not forced to conform to a one-size-fits-all solution.
+Loosely coupled teams enjoy the independence of selecting tools that best meet their specific needs. This tailored approach can lead to increased productivity and innovation, as teams are not forced to conform to a one-size-fits-all solution.
 
 ### [Customer Feedback](/capabilities/customer-feedback.md) - Related
 
-Rapid, independent deployments allow teams to incorporate customer feedback more effectively. While not a direct prerequisite, a culture of loosely coupled teams aligns well with feedback loops that drive continuous improvement and product-market fit.
+Rapid, independent deployments allow teams to incorporate customer feedback more effectively. While not a direct prerequisite, a culture of loosely coupled teams aligns well with regular feedback loops that drive continuous improvement and product-market fit.
 
 ### [Visibility of Work in the Value Stream](/capabilities/visibility-of-work-in-the-value-stream.md) - Upstream
 
 Clear visibility of work in the value stream is an essential precursor to establishing loosely coupled teams. By understanding how work flows through the organization, leaders can identify bottlenecks and design team structures that enhance autonomy and minimize dependencies.
 
 ### [Generative Organizational Culture](/capabilities/generative-organizational-culture.md) - Upstream
 
-A generative organizational culture fosters an environment where experimentation, innovation, and continuous learning are the norms. When teams are encouraged to question assumptions and generate new ideas, they’re better equipped to adopt loosely coupled practices and evolve independently while contributing to overall organizational resilience.
+A generative organizational culture fosters an environment where experimentation, innovation, and continuous learning are the norms. When teams are encouraged to question assumptions and generate new ideas, they’re more prone to adopt loosely coupled practices and evolve independently. Teams that are both inventive and independent contribute to overall organizational resilience.
```

Comment:
Let's just nix the last sentence. I know I wrote the resilience bit. But re-reading it, I'm not sure it adds anything all that valuable.